._travis.yml

```diff
@@ -10,6 +10,7 @@ script:
   - source sourceme.sh
   - cd src/test
   - pip install mypy
-  - ./mypy_with_exclusions.sh
+# Travis' mypy install causes some errors
```

Comment:
For the record I believe I see these errors as well thats why I made the other changes. So its some version issue that would appear for others.

Reply:
Ah good point, added the version number for mypy.